springserve/__init__.py

```diff
@@ -120,21 +131,57 @@ def API(reauth=False):
     return _API
 
 
+def V1API(reauth=False):
```

Comment:
For ClearLine Deals are only available in the V1 SpringServe API in addition to the current use of the V0 API here in the SDK. We currently include the API version in `base_url` we pass to link and link accesses a `/auth` url relative to `base_url` when it wants to authenticate to the API. 

* I don't know if updating link would break some other code which relies on it. Here I'm initializing separate instances of link's configuration for each API version. If there's a better approach, feedback is welcome.
* `_CONFIG_OVERRIDE` can currently override `base_url` and I'm not sure how this should be adapted to allow access to both the V0 and V1 API endpoints.